assets/css/webview_styles.css

```
@@ -327,11 +397,15 @@ svg {
 #darkModeToggle svg {
   fill: var(--app-on-primary) !important;
 }
+.theme-toggle svg {
+  fill: currentColor !important;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)